index.html

```diff
@@ -12,94 +12,124 @@
+
+				<!-- ## COUNTDOWN TIMER ## -->
+				<div id="qcCountdown" class="clearfix">
+					<!-- ## DAYS ## -->
+					<div class="dash days_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+							<div class="digit digit-3">0</div>
+						</div>
+						<div class="dash_title"><span>days</span></div>
 					</div>
-					<div class="dash_title"><span>days</span></div>
-				</div>
-				<!-- ## HOURS ## -->
-				<div class="dash hours_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## HOURS ## -->
+					<div class="dash hours_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>hours</span></div>
 					</div>
-					<div class="dash_title"><span>hours</span></div>
-				</div>
-				<!-- ## MINUTES ## -->
-				<div class="dash minutes_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## MINUTES ## -->
+					<div class="dash minutes_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>minutes</span></div>
 					</div>
-					<div class="dash_title"><span>minutes</span></div>
-				</div>
-				<!-- ## SECONDS ## -->
-				<div class="dash seconds_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## SECONDS ## -->
+					<div class="dash seconds_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>seconds</span></div>
 					</div>
-					<div class="dash_title"><span>seconds</span></div>
 				</div>
+				<!-- ## COUNTDOWN TIMER END ## -->
+				<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video">Buy Tickets</a>
 			</div>
-			<!-- ## COUNTDOWN TIMER END ## -->
-
-			<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video">Buy Tickets</a>
-
 		</div>
-
 	</section>
-	<!-- ## FULLSCREEN SLIDES ## -->
 
+	<!-- ## SPONSOR SECTION ## -->
+	<section class="qcContainer sponsors-section" id="sponsors">
+	  <div class="row middle-md">
+	    <div class="col-xs-12 col-md-6">
+	      <h2>Our Sponsors</h2>
+	    </div>
+	    <div class="col-xs-12 col-md-6 end-md">
+	      <a href="sponsor-prospectus.pdf" target="_blank" class="prospectus-link">Interested in becoming a sponsor?</a>
+	    </div>
```

Comment:
i just noticed that the button in our top section is very rounded, and this button is less rounded. the template isn't consistent but i think we should be. would you mind picking a button-roundedness and changing the existing buttons to that? i'll defer to your judgement on how rounded we should be ;)